dns-controller/cmd/dns-controller/main.go

```diff
@@ -95,7 +96,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	dnsController, err := dns.NewDNSController(dnsProvider, zoneRules)
+	dnsController, err := dns.NewDNSController(dnsProvider, zoneRules, dnsProviderId)
```

Comment:
This is decided by the --dns flags inside DAEMON_ARGS. So it's setting to coredns at here: https://github.com/vmware/kops/pull/6/files/6326f161619435794c37157b7ef7c79db636ea1b#diff-b25ed23c3990586a8ed6e64c73bb178cR276